tests/test_transforms/test_embeddings/test_models/test_tstcc.py

```diff
@@ -144,3 +153,40 @@ def test_logged_loss(ts_with_exog_nan_begin_numpy, verbose, n_epochs, n_lines_ex
     model.fit(ts_with_exog_nan_begin_numpy, n_epochs=n_epochs, verbose=verbose)
     check_logged_loss(log_file=file.name, n_lines_expected=n_lines_expected)
     tslogger.remove(idx)
+
+
+def test_correct_list_models():
+    assert TSTCCEmbeddingModel.list_models() == ["tstcc_medium"]
+
+
+@pytest.mark.parametrize("model_name", ["tstcc_medium"])
+def test_load_pretrained_model_default_path(model_name):
+    path = Path.home() / ".etna" / "embeddings" / "tstcc" / f"{model_name}.zip"
+    _ = TSTCCEmbeddingModel.load(path=path, model_name=model_name)
+    assert os.path.isfile(path)
+
+
+@pytest.mark.parametrize("model_name", ["tstcc_medium"])
+def test_load_pretrained_model_exact_path(model_name, tmp_path):
+    path = Path(tmp_path) / "tmp.zip"
+    _ = TSTCCEmbeddingModel.load(path=path, model_name=model_name)
+    assert os.path.isfile(path)
+
+
+def test_load_unknown_pretrained_model():
+    with pytest.raises(NotImplementedError):
```

Comment:
We should probably check the error message by `match`.

Reply:
fixed

Reply:
It was only fixed for one model. It should be fixed for both.